pkg/router.go

```diff
@@ -4,19 +4,26 @@ import (
 	"fmt"
 	"io"
 	"sync"
+	"sync/atomic"
+	"time"
 
 	"github.com/pion/ion-sfu/pkg/log"
 	"github.com/pion/rtcp"
 	"github.com/pion/webrtc/v3"
 )
 
+const ()
+
+var routerConfig RouterConfig
```

Comment:
A global router config was added to not pass the same configuration through many functions.